model/aro_hcp/v1alpha1/UpgradePolicyStateValue.model

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Nit: I noticed this file's name uses camel case as opposed to snake case, since it is now duplicated - any chance we can change that for consistency?